.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
- [ ] Please fix this error that appears in your project.
![image](https://user-images.githubusercontent.com/74744787/229234563-e02d52e9-b279-48a2-94ce-a48bbc5bc8b5.png)

Reply:
can I have an indication on how to solve the rpoblem please?

Reply:
can I have an indication on how to solve the problem please?

Reply:
Hi @Khanetmoi I think The "airbnb" configuration is outdated: If the "airbnb" configuration has been updated but you haven't updated your local version of it, you may encounter this error. To fix it, you can try running `npm update eslint-config-airbnb `